src/test/java/com/pgs/booking/controller/PassengerControllerTest.java

```diff
@@ -62,14 +59,12 @@
     public void setUP() {
         mockMvc = MockMvcBuilders
                 .webAppContextSetup(wac)
-                .apply(springSecurity())
                 .build();
     }
 
-    @WithMockUser
+    @SneakyThrows
```

Comment:
Some methods use @SneakyThrows and some just have throws declaration. Pick one approach and stick to it.